controllers/globals.go

```diff
@@ -0,0 +1,5 @@
+package controllers
+
+import "github.com/rh-ecosystem-edge/kernel-module-management/internal/syncronizedmap"
+
+var syncronizedKernelOsDtkMapping = syncronizedmap.NewSyncronizedKernelOsDtkMapping()
```

Comment:
I think this should be an instance variable of all 3 reconcilers.

Reply:
https://github.com/rh-ecosystem-edge/kernel-module-management/pull/96#discussion_r1020243279
This is what I think :)

Reply:
i agree with @qbarrand : this can be initialized in the main and passed as parameter during reconcilers'initialization

Reply:
Fixed.